packages/angular/src/executors/module-federation-dev-server/module-federation-dev-server.impl.ts

```diff
@@ -39,7 +40,6 @@ export async function* moduleFederationDevServerExecutor(
   // Force Node to resolve to look for the nx binary that is inside node_modules
   const nxBin = require.resolve('nx/bin/nx');
   const options = normalizeOptions(schema);
-  options.staticRemotesPort ??= options.port + 1;
```

Comment:
Removed this because it needs to be defaulted from the results of `getRemotes` (just like in React). Otherwise there could be a port conflict if one of the dev remotes has a port that's host port + 1.